src/main/java/com/onarandombox/MultiverseCore/commands/CreateCommand.java

```diff
@@ -0,0 +1,110 @@
+import org.bukkit.plugin.Plugin;
+import org.jetbrains.annotations.NotNull;
+
+@CommandAlias("mv")
+public class CreateCommand extends MultiverseCommand {
+
+    public CreateCommand(@NotNull MultiverseCore plugin) {
+        super(plugin);
+
+        this.plugin.getCommandManager().getFlagsManager().registerFlagGroup(FlagGroup.builder("mvcreate")
+                .add(MVValueFlag.builder("--seed", String.class)
+                        .addAlias("-s")
+                        .completion(() -> Collections.singleton(String.valueOf(new Random().nextLong())))
+                        .build())
+                .add(MVValueFlag.builder("--generator", String.class)
+                        .addAlias("-g")
+                        .completion(() -> Arrays.stream(Bukkit.getServer().getPluginManager().getPlugins())
+                                .filter(Plugin::isEnabled)
+                                .filter(genplugin -> this.plugin.getUnsafeCallWrapper().wrap(
+                                        () -> genplugin.getDefaultWorldGenerator("world", ""),
+                                        genplugin.getName(),
+                                        "Get generator"
+                                ) != null)
+                                .map(genplugin -> genplugin.getDescription().getName())
+                                .collect(Collectors.toList()))
+                        .build())
+                .add(MVValueFlag.builder("--world-type", WorldType.class)
+                        .addAlias("-t")
+                        .context((value) -> {
+                            try {
+                                return WorldType.valueOf(value.toUpperCase());
+                            } catch (IllegalArgumentException e) {
+                                throw new CommandException("Invalid world type: " + value);
+                            }
+                        })
+                        .completion(() -> {
+                            List<String> types = new ArrayList<>();
+                            for (WorldType type : WorldType.values()) {
+                                types.add(type.name().toLowerCase());
+                            }
+                            return types;
+                        })
+                        .build())
+                .add(MVFlag.builder("--adjust-spawn")
+                        .addAlias("-n")
+                        .build())
+                .add(MVFlag.builder("--no-structures")
+                        .addAlias("-a")
+                        .build())
+                .build());
+    }
+
+    @Subcommand("create")
+    @CommandPermission("multiverse.core.create")
+    @CommandCompletion("WORLDNAME  @flags:groupName=mvcreate")
+    @Syntax("<name> <env> -s [seed] -g [generator[:id]] -t [worldtype] [-n] -a [true|false]")
+    @Description("") //TODO
+    public void onCreateCommand(CommandIssuer issuer,
+
+                                @Flags("trim")
```

Comment:
What is this?

Reply:
Ahh something i copied from my old ACF branch, it will be passed to the Context resolver to input sanitize the worldname. (which will be implemented soon)

Reply:
If `"trim"` is suppose to be a particular option we should probably make it not a string.

Reply:
yea im removing it first, will work on it seperately